src/solvers/sat/satcheck.cpp

```diff
@@ -22,3 +23,14 @@ Author: Daniel Kroening, kroening@kroening.com
 #error "I expected to have MiniSat 2"
 #endif
 #endif
+
+satcheck_infot parse_satcheck_info(const optionst &options)
+{
+  satcheck_infot satcheck_info;
+  if(!options.get_option("random-seed").empty())
+    satcheck_info.random_seed = std::stoul(options.get_option("random-seed"));
```

Comment:
Use `safe_string2size_t` as `sizeof(unsigned long)!=sizeof(std::size_t)` on Windows.

Reply:
Good catch. But why not `stoull` (`unsigned long long`)?

Reply:
Because that wouldn't be correct on 32-bit architectures as `sizeof(unsigned long long)!=sizeof(std::size_t)` on those?